roofit/roofitcore/inc/RooVectorDataStore.h

```diff
@@ -62,6 +62,7 @@ class RooVectorDataStore : public RooAbsDataStore {
 
   /// Output struct for the RooVectorDataStore::getArrays() helper function,
   /// meant to be used for RooFit internal use.
+  /// @{
```

Comment:
By now I think that this doesn't work. Better use
```
/// \name ...
/// ...
/// @{

/// Output struct for the RooVectorDataStore::getArrays() helper function,
/// meant to be used for RooFit internal use.
/// struct ArraysStruct {
...
};

/// @}

Reply:
Ok I understood now how Doxygen works here (see https://www.doxygen.nl/manual/grouping.html).

I managed to group the two internal functions `getArrays` and `recomputeSumWeight` now, but unfortunately it was not possible to group the `struct ArraysStruct` together with the member functions. I had to leave it out of the group and note in the class documentation that this is internal.

I also built the doxygen myself not to verify that `getArrays` and `recomputeSumWeight` go nicely in the "Internal RooFit interface" group.